src/treesitter/mod.rs

```diff
@@ -58,19 +33,9 @@ pub fn get_results(
                 .collect::<Vec<_>>()
         })
         .map(|node| {
-            let point = node.start_position();
-            Result {
-                point,
-                line_text: {
-                    let file = File::open(file_path).unwrap();
-                    BufReader::new(file)
-                        .lines()
-                        .nth(point.row)
-                        .unwrap()
-                        .unwrap()
-                },
-                file_path: file_path.to_owned(),
-            }
+            let range = node.range();
+
+            Match::new(range.start_byte, range.end_byte)
```

Comment:
Directly use the `Match` struct, even though the start and end numbers are potentially wrong (see my comment above about `at`).